protos/feast/core/CoreService.proto

```diff
@@ -432,4 +453,50 @@ message UpdateFeatureSetStatusRequest {
     FeatureSetStatus status = 2;
 }
 
-message UpdateFeatureSetStatusResponse {}
\ No newline at end of file
+message UpdateFeatureSetStatusResponse {}
+
+message ApplyFeatureTableRequest {
+    // Optional. Name of the Project to apply the Feature Table to.
+    // If unspecified, will apply FeatureTable to the default project.
+    string project = 1;
+    // Feature Table specification to apply
+    FeatureTableSpec table_spec = 2;
+}
+
+message ApplyFeatureTableResponse {
+    FeatureTable table = 1;
+}
+
+message GetFeatureTableRequest {
+    // Optional. Name of the Project to retrieve the Feature Table from.
+    // If unspecified, will apply FeatureTable to the default project.
+    string project = 1;
+
+    // Name of the FeatureTable to retrieve.
+    string name = 2;
+}
+
+message GetFeatureTableResponse {
+    // The Feature Table retrieved.
+    FeatureTable table = 1;
+}
+
+message ListFeatureTablesRequest {
```

Comment:
why we dropped wildcard search? it was used in JobController to retrieve all feature-sets by store subscription and I guess it could be useful for JobService as well if we won't drop subscription mechanism in stores.

Reply:
@woop will we support several online stores with different subscriptions in new architecture?

Reply:
So you mean we should support single project, wildcard feature table name matches?

Reply:
We only have a single online store. There is no such thing as a store subscription any more.

Reply:
When we implement a `stream sink` in the future, then we will need to create some kind of `ingestion process` that hooks up to a stream based on configuration to update multiple online stores. But Feast doesnt know about these in the form of subscriptions. There is also no call to Feast Core to get this information.